src/components/signup/SmsForm.js

```diff
@@ -133,12 +131,13 @@ class SmsForm extends React.Component<Props, State> {
           <Section.Stack justifyContent="center">
             <OtpInput
               shouldAutoFocus
-              numInputs={this.numInputs}
+              numInputs={NumInputs}
               onChange={this.handleChange}
               hasErrored={errorMessage !== ''}
               errorStyle={styles.errorStyle}
               value={otp}
               placeholder="*"
+              isInputNum={true}
```

Comment:
Instead of specifying *isInputNum*, I added *keyboardType* prop, because I remember there was an error with numeric inputs here... but maybe it is fixed now with the pattern validation I added.
We can use keyboardType="phone-pad" and it should work exactly the same, but using a text input

Reply:
if isInputNum=true then it uses "phone-pad", the keyboardType was redundant